docker/docker_client.go

```diff
@@ -285,8 +287,11 @@ func CheckAuth(ctx context.Context, sys *types.SystemContext, username, password
 	if err != nil {
 		return errors.Wrapf(err, "error creating new docker client")
 	}
-	client.username = username
-	client.password = password
+
```

Comment:
(Absolutely non-blocking: There wasn’t a newline here because `newDockerClient` requires callers to set the credentials separately; that’s conceptually a single operation.)